src/zns.ts

```diff
@@ -99,8 +94,7 @@ export default class Zns extends NamingService {
     const resolution = this.structureResolverRecords(
       await this.getResolverRecords(resolverAddress),
     );
-
-    const addresses = _.mapValues(resolution.crypto || {}, 'address');
+    const addresses = mapValues(resolution.crypto || {}, 'address');
```

Comment:
I don't think we need a function this complex. `Object.values(resolution.crypto).map(c => c.address)` would work just fine.